### ISSUE TITLE
OVR headset device does not work with the latest version of yarp

### ISSUE THREAD

**Issue by GiulioRomualdi** [robotology/yarp#2103]
I updated yarp to the latest version of devel and the device of the  `ovr headset` is not running.
We are using the yarp in `devel` and Oculus SDK version is `1.40.0`. (is there a particular version of the sdk that should be used?)
This the log file for the error:
[log_icub-virtualizer_yarpdev_7848.txt](https://github.com/robotology/yarp/files/3698309/log_icub-virtualizer_yarpdev_7848.txt)

This is what I can see when I connect the ports.
![ovr_headset](https://user-images.githubusercontent.com/16744101/66327184-b1a83580-e92a-11e9-8c42-79d40b0c170e.png)

cc @aerydna @S-Dafarra @kouroshD @DanielePucci @Nicogene

**Comment by GiulioRomualdi** [comment]
Looking at the code I discovered that the error
```
<ERROR> FrameTransformClient::getTransform() frames  mobile_base_body_link  and  head_link  are not connected 
```
is triggered only when the `relative` mode is enabled in the `ovrheadset` device.

In a previous version of yarp the `relative` mode was disabled.
https://github.com/robotology/yarp/commit/b2800f222352cd0cc410f6826269d432460b691a#diff-512556b5de879621a97ebd62f0575b8aL290

Now the `relative` flag is not set so the behavior of the code may be unpredictable.
What do you think about passing the `relative` parameter from configuration?

**Comment by GiulioRomualdi** [comment]
This commit https://github.com/GiulioRomualdi/yarp/commit/aef216e33857c077d16cdd264510e192f439462c should fix the bug

**Comment by drdanz** [comment]
@GiulioRomualdi Thanks for investigating this. Can you please open a pull request proposing these changes on `master`?

@aerydna Can you please have a look at the suggested changes?

**Comment by aerydna** [comment]
ok for me!

**Comment by drdanz** [comment]
Fixed by #2106
